test/unit/talkToBackendService.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('TalkToBackendService', function(){
+  beforeEach(module('playfestApp'));
+
+  var TalkToBackendService, httpBackend;
+
+  var festivalData = ["Glastonbury", "Glastonburyed"];
+  var confirmationData = "Your playlist has been created!"
+
+  beforeEach(inject(function(_TalkToBackendService_, $httpBackend){
+    TalkToBackendService = _TalkToBackendService_;
+    httpBackend = $httpBackend;
+  }));
+
+  describe('#getFestivalMatches', function(){
+    it('gets back potential festival matches', function(){
+      httpBackend.expectGET("https://localhost/5000/festivalMatches/Sommerset").respond(festivalData);
+
+      TalkToBackendService.getFestivalMatches("Sommerset").then(function(festivals){
+        expect(festivals).toEqual(["Glastonbury", "Glastonburyed"]);
```

Comment:
change the hardcoded ["Glastonbury", "Glastonburyed"] to the var we're passing to the mock (festivalData)